chef/cookbooks/nova/recipes/config.rb

```diff
@@ -329,7 +316,6 @@
             :keystone_settings => keystone_settings,
             :cinder_insecure => cinder_insecure || keystone_settings['insecure'],
             :ceph_user => ceph_user,
```

Comment:
I think the user bit will have the same issue as the uuid bit: it should come from cinder, so we should not put it nova.conf.

About @dirkmueller's point that we need it for ephemeral storage: this is correct, but I guess we could figure it out when working on the ephemeral bit. Could be an argument for not dropping the attributes from the schema for now, though. (But I'd still drop them from nova.conf)